src/_tutorials/server/cmdline.md

```diff
@@ -20,17 +20,16 @@ prevpage:
   * Most input and output requires the use of streams.
 </div>
 
-<aside class="alert alert-info" markdown="1">
-<strong>Prerequisite:</strong>
-This tutorial uses the `async` and `await` language features, which rely on the
-<a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/Future-class.html" target="_blank">Future</a>
-and
-<a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/Stream-class.html"
-   target="_blank">Stream</a> classes for asynchronous support.
-To learn more about these features, see the
-[asynchronous programming codelab](/codelabs/async-await) and the
-[streams tutorial](/tutorials/language/streams).
-</aside>
+{{site.alert.note}}
```

Comment:
@kwalrath This aside used to have a _Prerequisite_ inlined heading, but the note text isn't asking the reader to do anything. I've changed this to have a standard **Note** heading (with the usual icon). Maybe this should be an **Important** note instead? If you'd like to keep _Prerequisite_ as a heading, then maybe the text should be tweaked to say something like "ensure that your `PATH` is appropriately setup", but possibly just a **Note** is enough?